src/ngx_http_lua_shdict.c

```diff
@@ -401,7 +425,11 @@ ngx_http_lua_shdict_get_helper(lua_State *L, int get_stale)
                           "but only seen %d", n);
     }
 
-    zone = lua_touserdata(L, 1);
+    if (lua_type(L, 1) != LUA_TTABLE ) {
```

Comment:
Style: extra space before the `)` character. I'll fix it locally during the merge anyway.